tests/test_main.py

```diff
@@ -151,11 +156,14 @@ def test_sigterm_enduring_bare():
 
     async def main():
         shutdown_waits_for(corofn())  # <-- Look Ma! No awaits!
+    try:
+        kill(SIGTERM, after=0.01)
+        newloop()
+        run(main())
+        assert items
+    except KeyboardInterrupt:
```

Comment:
Under what circumstances does a `KeyboardInterrupt` get raised in the above code? 

I'm guessing that you added this because maybe sometimes the shutdown doesn't actually occur and the loop never shuts down, and so you wanted a way to manually stop it...?

Reply:
@cjrh I was having a problem where CI simply terminates: https://travis-ci.org/cjrh/aiorun/jobs/511961630#L244